src/runtime_src/core/pcie/driver/linux/xocl/subdev/mailbox.c

```diff
@@ -1951,18 +1951,43 @@ static int mailbox_offline(struct platform_device *pdev)
 	 */
 	mailbox_disable_intr_mode(mbx, false);
 #else
-	mailbox_disable_intr_mode(mbx, true);
+	/* stop polling */
```

Comment:
this maybe not safe. if someone send out a msg at this moment, timer will be activated, still...